docs/source/outputs.rst

```diff
@@ -0,0 +1,172 @@
+.. SPDX-FileCopyrightText: ASSUME Developers
+..
+.. SPDX-License-Identifier: AGPL-3.0-or-later
+
+###############
+Storing Outputs
+###############
+
+This documentation describes the functionality of the output processing system used to store simulation outputs either in a database or as CSV files. It covers the configuration, data buffering, conversion, and storage mechanisms of the main class as well as the support class for database maintenance.
+
+
+WriteOutput Class
+==================
+
+The ``WriteOutput`` class is responsible for:
+
+- Collecting simulation output messages.
+- Buffering data until either a time-based interval or a memory limit is reached.
+- Converting raw data into structured pandas DataFrames.
+- Storing data into a database and/or exporting it as CSV files.
+- Managing performance trade-offs by allowing configuration of buffer sizes and storage intervals.
+
+Constructor Parameters
```

Comment:
Having API docs here as well, makes things redundant and harder to maintain.
I would rather just link to the API docs of the WriteOutput class, which currently is here:
https://assume--569.org.readthedocs.build/en/569/assume.common.html#module-assume.common.outputs